Problem1.java

```diff
@@ -0,0 +1,54 @@
+class MyQueue {
```

Comment:
Kindly mention the time complexity for all operations. Also mention Space complexity.